src/parser.cpp

```diff
@@ -6337,15 +6402,38 @@ gb_internal bool parse_file(Parser *p, AstFile *f) {
 
 	String filepath = f->tokenizer.fullpath;
 	String base_dir = dir_from_path(filepath);
-	if (f->curr_token.kind == Token_Comment) {
-		consume_comment_groups(f, f->prev_token);
+
+	Array<Token> tags = array_make<Token>(ast_allocator(f));
+
+	bool first_invalid_token_set = false;
+	Token first_invalid_token = {};
+
+	while (f->curr_token.kind != Token_package && f->curr_token.kind != Token_EOF) {
```

Comment:
Since we need to find the file tag tokens and skip comment groups we now loop until we find the package token.